recipes/libs/fontconfig.yaml

```diff
@@ -0,0 +1,46 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+    PKG_VERSION: "2.13.92"
+
+depends:
+    - name: devel::gettext
+      use: [result, deps, tools]
+      tools:
+          target-toolchain: host-toolchain
```

Comment:
i that the way we do it in future? 
when we add tools to the rootrecipe, when we do it like this?

Reply:
There is no hard rule but I would put it this way: if a tool is used often, especially in a class, it should be added to the rootrecipe class. On the other end of the spectrum, if a tool is only used for special purposes, then the tool remapping should be used. Of course there is a grey area in between.